routes/users.js

```diff
@@ -1021,14 +1007,15 @@ router.get('/user/:id/activate', async function(req, res) {
             return plugin_call(plugin_info, user.uid, user, session_user.uid);
             // eslint-disable-next-line no-unused-vars
         })).then(function(results){
-            return send_notif(mailOptions, fid, []);
+            res.send({msg: 'Activation in progress', fid: fid, error: []});
+            res.end();
+            return;
         }, function(err){
-            return send_notif(mailOptions, fid, err);
-        }).then(function(errs){
```

Comment:
mail is sent before plugin, in case of err, won't be notified of error

Reply:
error was not sent by mail in previous version, like "fid", "err" was only pass as parameter to be used in the next ".then" (ie callback)